src/datasets/experiment_monitoring.md

```diff
@@ -0,0 +1,60 @@
+
+| Dataset name                                                                             | Description                                                         |
+| ---------------------------------------------------------------------------------------- | ------------------------------------------------------------------- |
+| `moz-fx-data-shared-prod.telemetry.experiment_unenrollment_overall`                      | Overall number of clients that unenrolled from experiments          |
+| `moz-fx-data-shared-prod.telemetry.experiment_enrollment_other_events_overall`           | Number of events other than `enroll` and `unenroll` sent by clients |
+| enrolled in experiments                                                                  |
+| `moz-fx-data-shared-prod.telemetry.experiment_enrollment_cumulative_population_estimate` | Cumulative number of clients enrolled in experiments                |
+| `moz-fx-data-shared-prod.telemetry.experiment_enrollment_overall`                        | Overall number of clients enrolled in experiments                   |
+| `moz-fx-data-sharedo-prod.telemetry.experiment_enrollment_daily_active_population`       | Number of daily active clients enrolled in experiments              |
+
+## Experiment search metrics data
+
+`moz-fx-data-shared-prod.telemetry_derived.experiment_search_aggregates_live_v1` provides aggregated search metrics of clients enrolled in experiments, such as the number of searches performed, the number of searches with ads and the number of ad clicks. This live view is also the basis of several derived tables:
+
+| Dataset name                                                                    | Description                                                               |
+| ------------------------------------------------------------------------------- | ------------------------------------------------------------------------- |
+| `moz-fx-data-shared-prod.telemetry.experiment_cumulative_ad_clicks`             | Cumulative number of ad clicks by clients enrolled in experiments         |
+| `moz-fx-data-shared-prod.telemetry.experiment_cumulative_search_count`          | Cumulative number of searches by clients enrolled in experiments          |
+| `moz-fx-data-shared-prod.telemetry.experiment_cumulative_search_with_ads_count` | Cumulative number of searches with ads by clients enrolled in experiments |
+
+## GCS data export
+
+As some dashboard solutions, such as the Experimenter console, might not have access to BigQuery, data from derived experiment monitoring tables is also exported as JSON to `monitoring/` in the `mozanalysis` bucket in `moz-fx-data-experiments`. JSON files are named like: `<experiment_slug>_<monitoring_dataset_name>.json`.
```

Comment:
It may be worth giving a full example path to make this more concrete, like:

    gs://mozanalysis/monitoring/bug-1683348-rollout-tab-modal-print-ui-roll-out-release-84-85_experiment_unenrollment_overall.json